plugins/rest/plugin_impl_rest.go

```diff
@@ -70,6 +76,10 @@ type Deps struct {
 	VPP          vpp.API
 }
 
+type indexes struct {
+	IdxMap map[string][]indexItem
```

Comment:
This should be called `ItemMap`, it doesn't have much to do with index, other than it's served on index page.